Tests/PackageGraphPerformanceTests/DependencyResolverPerfTests.swift

```diff
@@ -120,6 +120,74 @@ class DependencyResolverPerfTests: XCTestCase {
             }
         }
     }
+
+    func testKitura() throws {
+        let graph = try mockGraph(for: "kitura.json")
+        let provider = MockPackagesProvider(containers: graph.containers)
+
+        measure {
+            let resolver = MockDependencyResolver(provider, MockResolverDelegate())
+            let result = try! resolver.resolve(constraints: graph.constraints)
+            graph.checkResult(result)
+        }
+    }
+
+    func testZewoHTTPServer() throws {
```

Comment:
Can you factor out the boilerplate here, so each instance is just one call, which passes the input file name?